### PR TITLE
SOC-1476 | Translations for discussions moved to separated files

### PR DESCRIPTION
**Pull request by Bloodmoon24**
https://wikia-inc.atlassian.net/browse/SOC-1476

@bkoval @kvas-damian please CR

### CODE REVIEW

File: front/locales/es/discussion.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "back-to-wikia-link":"A la wikia",
+    "number-of-posts":"__numberOfPosts__ publicaciones",
+    "connection-error":"No se puede conectar, por favor inténtalo de nuevo.",
+    "not-found-error":"¡Uh oh, parece que esta página no existe!",
+    "action-retry-title-desktop":"Haz clic para volver a intentar",
+    "action-retry-title-mobile":"Toca para volver a intentar",
+    "all-discussions-link-desktop":"Muéstrame todas las discusiones",
+    "all-discussions-link-mobile":"Todas las discusiones"
+  }
+}
```
Comment (bkoval) [inline front/locales/es/discussion.json]: Add new lines at the end if discussion.json files as I did with the first one. Also, you can set up your IDE to add a new line for you :)

**Comment by bkoval** [comment]
I can see that translation messages are moved, by their references in code are not changed. Did you check if the messages still show up as they did before? :)

**Comment by Warkot** [comment]
Have you gave a thought on how to update Crowdin after these changes?

File: front/scripts/main/routes/DiscussionPostRoute.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ App.DiscussionPostRoute = Em.Route.extend(App.UseNewNavMixin, App.DiscussionRout
 	afterModel (model: typeof App.DiscussionPostModel): void {
 		var title: string = model.get('title');
 		if (!title) {
-			title = i18n.t('discussion.share-default-title', {siteName: Mercury.wiki.siteName});
+			title = i18n.t('discussion:menu.share-default-title', {siteName: Mercury.wiki.siteName});
```
Comment (rogatty) [inline front/scripts/main/routes/DiscussionPostRoute.ts]: Shouldn't this be `discussion:main.share-default-title`? The same in https://github.com/Wikia/mercury/pull/1557/files#diff-b464d399d11d6925df262840329ca34cR56

**Comment by bkoval** [comment]
@Warkot @rogatty Thanks for the comments! The work here is still in progress, I added the appropriate label.

@Warkot - What do you mean by "breaking stuff"? In the best case scenario we would like to move translations for discussions for all supported languages to a separate file, then upload them all with the "-t" option in the crowdin tools. Can this cause any damage in crowdin? If so, is there any safe path we can follow here?

**Comment by Warkot** [comment]
Just be sure that you're updating the most recent translations, to not to overwrite/remove any existing ones, which may have appeared after branching from `dev`. Downloading the translations from Crowdin and merging them with changes above should do the trick. To be sure, you might also merge current `dev` here.

**Comment by bkoval** [comment]
@Warkot That's an awesome tip, thanks! @Bloodmoon24 let's make sure that before submitting it to crowdin:

 - we merged the newest version of dev branch to this one
 - we fetched newest translations from crowdin and pushed them along with changes from the pull request

**Comment by Bloodmoon24** [comment]
Thanks for feedback. As I can see this pull request was to early created, sorry for that. @bkoval  Thanks for label. @Warkot I going to contact you if I'll have any problems

**Pull request closed without merging**